index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>UI Challenge</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="styles/style.css" />
+</head>
+<body>
+  <h1>UI Challenge - Custom Form Elements</h1>
+  <form>
+    
+    <p>text input</p>
+    <input type="text" value="placeholder text">

```

Comment:
You should make this the `placeholder` rather than the `value`, so that it goes away when I start typing.